Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -228,7 +228,6 @@ @interface MessagesViewController () <AVAudioPlayerDelegate,
 @property (nonatomic) NSCache *messageAdapterCache;
 @property (nonatomic) BOOL userHasScrolled;
 @property (nonatomic) NSDate *lastMessageSentDate;
-@property (nonatomic) NSTimer *scrollLaterTimer;
```

Comment:
This was an earlier hack to work around https://github.com/WhisperSystems/JSQMessagesViewController/pull/17.  Now that we've addressed the root cause, this timer can go away!